db/schema.rb

```diff
@@ -111,6 +111,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "llms", force: :cascade do |t|
+    t.string "name"
+    t.text "description"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
Did you mean to add this table? I suspect this is from another branch. You can just revert this whole file.

Reply:
Fixed